recipe/meta.yaml

```diff
@@ -18,7 +20,7 @@ requirements:
 
   run:
     - python
-    - gitdb >=0.6.2
+    - gitdb >=0.6.4
```

Comment:
This got changed in commit ( https://github.com/gitpython-developers/GitPython/commit/a2d39529eea4d0ecfcd65a2d245284174cd2e0aa ), but as noted in the comments the `requirements.txt` file was not updated. That has since been corrected, but was not in this release. In any event, it is clear this change was intended for the patch release.